packages/react-components/package.json

```diff
@@ -51,6 +51,7 @@
     "@fluentui/react-popover": "9.0.0-beta.4",
     "@fluentui/react-portal": "9.0.0-beta.4",
     "@fluentui/react-provider": "9.0.0-beta.4",
+    "@fluentui/react-slider": "9.0.0-beta.4",
```

Comment:
I'm not seeing a change to actually export Slider (this just adds it as a dep). Also, Slider **shouldn't** be exported from the suite until we fix the primary slot issue. So it's best to just revert this change for now.

Reply:
changed